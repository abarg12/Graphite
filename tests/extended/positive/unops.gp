### Testing unary operations
### [positive test]

printf(-12);
printf(-0.111);

int a = 2;
printf(a);
a = -a;
printf(a);

float b = 42.24;
printf(b);
b = -b;
printf(b);

printf(true);
bool c = true;
printf(c);
c = not c;
printf(c);