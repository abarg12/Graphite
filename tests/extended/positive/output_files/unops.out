-12
-0.111
2
-2
42.24
-42.24
1
1
0
